internal/prober/multihttp/script_test.go

```diff
@@ -342,6 +342,42 @@ func TestBuildBody(t *testing.T) {
 	}
 }
 
+func TestInterpolateBodyVariables(t *testing.T) {
```

Comment:
We can get tests to run a bit faster if we t.Parallel() all the things!

```suggestion
func TestInterpolateBodyVariables(t *testing.T) {
	t.Parallel()

```

Reply:
Ah true, thank you!